core/templates/pages/donate-page/donate-page.component.spec.ts

```diff
@@ -118,4 +119,16 @@ describe('Donate page', () => {
       }
     );
   });
+
+  it('should change learner tile in carousel', () => {
+    fixture.detectChanges();
+    const randomVal = Math.floor(Math.random() * 5);
+    const tile = '#tile-' + randomVal;
+    const nativeEle = fixture.debugElement.nativeElement;
```

Comment:
nativeElem (we use elem in other files, in general try to be consistent with the existing codebase)